### DISCUSSION TITLE
Can't figure ou why the dead zone dot isn't centered on the follow target

### DISCUSSION THREAD

**Discussion by monstropato** [ramokz/phantom-camera#333] [Question / Help Wanted]
When setting up the camera, the viewfinder shows the little dot perfectly where it should be.
![Captura de tela 2024-06-17 153027](https://github.com/ramokz/phantom-camera/assets/77409633/b8fb4664-55d0-4600-9b52-cd3cfd363c25)

But when running the game the dot never centers in the player, instead it keeps on this corner, no matter what. Any ideas why?
![Captura de tela 2024-06-17 153008](https://github.com/ramokz/phantom-camera/assets/77409633/8bc23bc3-35b3-4ec9-a168-97d12e073eed)

**Comment by ramokz** [comment]
Not sure!

My hunch would be that the player target gets instantiated somewhere else and then teleports to that position when the game starts, which can be triggered by things like setting the `top_level` property. But difficult to tell what is happening from a screenshot alone.

The dot can get stuck on the edges if the target is entirely out of view, which is a bug, so that's where my thought went anyway. There are other bugs with the `Framed Follow`, which I'm looking to resolve at some point.
